java-manta-client/src/main/java/com/joyent/manta/client/multipart/EncryptedMultipartManager.java

```diff
@@ -52,6 +52,12 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(EncryptedMultipartManager.class);
 
     /**
+     * Object used to reset the state of encryption/authentication in case
+     * a part needs to be reuploaded.
+     */
+    private static final EncryptionStateRecorder RECORDER = new EncryptionStateRecorder();
```

Comment:
`EncryptionStateRecorder` is a helper class and has no state of its own. This reference has been removed and the nature of the class has been clarified by newer commits (private constructor, only static methods)